boards/risc-v/esp32c3/common/src/esp32c3_board_mpu60x0_i2c.c

```diff
@@ -0,0 +1,124 @@
+/****************************************************************************
+ * boards/risc-v/esp32c3/common/src/esp32c3_board_mpu60x0_i2c.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+
+#include <stdio.h>
+#include <debug.h>
+
+#include <nuttx/arch.h>
+#include <nuttx/kmalloc.h>
+#include <nuttx/sensors/mpu60x0.h>
+#include <nuttx/i2c/i2c_master.h>
+
+#include "esp32c3_i2c.h"
+
+#include "esp32c3_board_mpu60x0_i2c.h"
+
+/****************************************************************************
```

Comment:
I think we can clean-up many unused sections here

Reply:
It seems necessary in nxstyle.json rules

Reply:
The public functions section is obviously needed ;) I was talking about defines, types and other empty sections

Reply:
Thanks. it's passed the check, but one cancelled after 359 min. timeout maybe?